components/chrissrogers-noop@master/.gitignore

```diff
@@ -0,0 +1,2 @@
+components
+build
```

Comment:
All of the component commits from here on down are due to the test suite dependencies being loaded in the repo root, which is committed. As a bonus, the specs will now run faster since it will not be necessary to download any dependencies during CI runs.